app/src/sandbox/prefixes/PrefixComponent/PrefixComponent.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './PrefixComponent.scss';
```

Comment:
use module imports to import styles into component:
import css from './PrefixComponent.scss';
and then in jsx: className={ css.prefixWrapper }